posts/index.ts

```diff
@@ -0,0 +1 @@
+export { router as postRouter } from './postRouter';
```

Comment:
I like your use of index files. As backends grow it can be very cumbersome and having all your routes here is a great workflow advantage